tests/unit/src/main/java/org/mule/tck/junit4/matcher/ValueMatcher.java

```diff
@@ -54,13 +74,28 @@ public static ValueMatcher valueWithId(String id) {
     return valueWithId(is(id));
   }
 
+  /**
+   * Creates a new instance of the {@link ValueMatcher}
+   *
+   * @param id of the {@link Value}
+   * @param strict defines if the {@link Value} should be completed and childs validated in order
+   * @return the new instance of {@link ValueMatcher}
+   */
+  public static ValueMatcher valueWithId(String id, boolean strict) {
```

Comment:
why not a new `strict(boolean)` method? I think it would be better since that attribute is optional.